sonic_installer/main.py

```diff
@@ -476,5 +476,15 @@ def rollback_docker(container_name):
 
     click.echo('Done')
 
+# verify reboot
+@cli.command('verify_reboot')
+def verify_reboot():
```

Comment:
Along with Qi's comment above, this command should also be named form clearly, e.g., `verify-next-image`.

Reply:
Fixed.